test/deploy.py

```diff
@@ -0,0 +1,486 @@
+#!/usr/bin/python3
+import os
+import sys
+import pwd
+import json
+import yaml
+import time
+import pylxd
+import string
+import random
+import subprocess
+
+class Cleaner:
+    def __init__(self, model_file_path:string) -> None:
+        if os.path.exists(model_file_path):
+            # init client
+            self.client = pylxd.Client()
+            self.clean(model_file_path)
+        else:
+            print("Model File {} does not exists.".format(model_file_path))
+
+    def clean(self, model_file_path:string) -> None:
+        '''Clean LXC objects mentioned in the model file.'''
+        with open(model_file_path, 'r') as model_file:
+            model = json.loads(model_file.read())
+
+        vm_name = model['vm_name']
```

Comment:
A model file is generated at the end of VM creation process, hence all steps need to go successfully; In which case the model file will always have the 4 keys: 'vm_name', 'profile', 'storage_pool' and 'volumes'. It can turn out to be problematic if a user decides to make a model file (manually) and pass it to the script.